src/Map/accessToken.js

```diff
@@ -0,0 +1,3 @@
+export default mapToken = 'pk.eyJ1IjoibWh1Z2dpbnMxMjY1IiwiYSI6ImNrY3RzaG1zZDIxbGwyd2xiYjA4Mm5vMmcifQ.DAx-tZndkUGJ93jAja2zgA'
```

Comment:
I don't think you wanted to have this in the PR?

Reply:
Yeah we aren't even using the map that was needed for so I'll go ahead a delete it. Thanks for catching that!

Reply:
The file has been deleted and ready to merge